sites/breadcrumbs/inc-breadcrumbs.html

```diff
@@ -1,25 +1,26 @@
-{%- unless page.pageclass contains "home" -%}
+{%- unless page.pageclass contains "home" or page.breadcrumbs == false -%}
```

Comment:
I wasn't able to test any of the changes I made to this include since GitHub Pages always sticks with wet-boew/GCWeb#master's version of this include instead of mine.

Reply:
Notes: We will need to documents that in the gcweb-jekyll

Reply:
In order to test your version, you will need to overwrite the _includes file.